activesupport/test/number_helper_test.rb

```diff
@@ -45,8 +45,10 @@ def exabytes(number)
       def test_number_to_phone
         [@instance_with_helpers, TestClassWithClassNumberHelpers, ActiveSupport::NumberHelper].each do |number_helper|
           assert_equal("555-1234", number_helper.number_to_phone(5551234))
+          assert_equal("555-1234", number_helper.number_to_phone(5551234, {:area_code => true}))
           assert_equal("800-555-1212", number_helper.number_to_phone(8005551212))
           assert_equal("(800) 555-1212", number_helper.number_to_phone(8005551212, {:area_code => true}))
+          assert_equal("+1-(800) 555-1212", number_helper.number_to_phone(8005551212, {:area_code => true, :country_code => 1}))
```

Comment:
On what do you agree with @vipulnsward? If you did read the body of this PR you would've known that it was me that brought up the issue of the (possibly) incorrect formatted phone number.